native.wasm.js

```diff
@@ -0,0 +1,4 @@
+const { parse, xxhashBase64Url } = require('./wasm-node/bindings_wasm.js');
+
+exports.parse = parse;
+exports.xxhashBase64Url = xxhashBase64Url;
```

Comment:
```suggestion
const { parse, xxhash_base64_url } = require('./wasm-node/bindings_wasm.js');

exports.parse = parse;
exports.xxhashBase64Url = xxhash_base64_url;
```
Currently, `wasm-pack` doesn't convert `under_score_case` to `camelCase`.

Reply:
Good catch! I felt renaming on the wasm side will make the code more consistent. So I did it in that way.

Reply:
Nice work! I wasn't aware of this attribute `js_name` before.